### PR TITLE
Fix/databend params

### PR DESCRIPTION
**Pull request by hantmac**
## What type of PR is this? 

- [x] Bug Fix

- Fix the params for databend query
- Add databend logo

## Description
<!-- In case of adding / modifying a query runner, please specify which version(s) you expect are compatible. --

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
a029ebf5-5fcc-41c8-98b9-a1a80a459c69

### CODE REVIEW

**Review by arikfr — APPROVED** [review]

File: redash/query_runner/databend.py [hunk 1]
```diff
@@ -33,11 +33,11 @@ def configuration_schema(cls):
 
     @classmethod
     def name(cls):
-        return "Databend"
+        return "databend"
```
[the remark this answers is not in the document]
Comment (hantmac) [reply redash/query_runner/databend.py:36]: LGTM!

**Review by hantmac — COMMENTED** [review]
LGTM

File: redash/query_runner/databend.py [hunk 2]
```diff
@@ -61,12 +61,12 @@ def _define_column_type(column_type):
             return TYPE_STRING
 
     def run_query(self, query, user):
-        host = (self.configuration.get("host") or "localhost"),
-        port = (self.configuration.get("port") or 8000),
-        username = (self.configuration.get("username") or None),
-        password = (self.configuration.get("password") or None),
-        database = (self.configuration.get("database") or None),
-        secure = (self.configuration.get("secure") or False),
+        host = self.configuration.get("host") or "localhost"
```
Comment (denisov-vlad) [inline redash/query_runner/databend.py:64]: `.get` method allows to set the default value:

`self.configuration.get("host", "localhost")`

Comment (arikfr) [reply redash/query_runner/databend.py:64]: Good point... I've been stuck in Typescript for a while, so much so that I overlooked this one 🤦‍♂️

Comment (arikfr) [reply redash/query_runner/databend.py:64]: I merged this PR anyway as it's still better than the current state, but @hantmac you're welcome to submit an update to the config fetching code.

Comment (hantmac) [reply redash/query_runner/databend.py:64]: Aha.. thanks very much. I would like to update to the config fetching code.

**Pull request merged**